testsuite/capabilities/providers.py

```diff
@@ -4,7 +4,7 @@
 from testsuite.configuration import openshift
 
 
-def _rhoam():
+def is_rhoam():
```

Comment:
this should be moved to dynaconf_loader.py where `data["threescale"]["deployment_type"]` should be set to one of `(onprem, rhoam, saas)`. Such solution has an advantage in posibility to overwrite it by user with `_3SCALE_TESTS_threescale__deployment_type=rhoam`

Reply:
@mganisin moved to dynaconf loader, for now, there is detection only for RHOAM, do you have any suggestions about how to detect saas environment?